docs/guides/result_storage.mdx

```diff
@@ -0,0 +1,95 @@
+
+In Crawlee, the key-value store is represented by the KeyValueStore class. To facilitate easy access to the default key-value store, Crawlee provides the functions `KeyValueStore.getValue()` and `KeyValueStore.setValue()`.
+
+The data is stored in the directory specified by the `CRAWLEE_STORAGE_DIR` environment variable as follows:
+```text
+{CRAWLEE_STORAGE_DIR}/key_value_stores/{STORE_ID}/{KEY}.{EXT}
+```
+- `{CRAWLEE_STORAGE_DIR}`: The root directory for all storage data specified by the environment variable.
+- `{STORE_ID}`: The identifier for the key-value store, either "default" or as specified by `CRAWLEE_DEFAULT_KEY_VALUE_STORE_ID`.
+- `{KEY}`: The unique key for the record.
+- `{EXT}`: The file extension corresponding to the MIME type of the content.
+
+The following code demonstrates basic operations of key-value stores:
+
+<Tabs groupId="keyvaluestore">
+    <TabItem value="kvstore_basic_example" label="Basic usage" default>
+        <CodeBlock className="language-python">
+            {KvStoreBasicExample}
+        </CodeBlock>
+    </TabItem>
+</Tabs>
+
+To see a real-world example of how to get the input from the key-value store, see the [Screenshots](https://crawlee.dev/python/docs/examples/capture-screenshots-using-playwright) example.
+
+## Dataset
+
+Datasets are utilized for storing structured data, where each object contains the same attributes, like products from an online store or listings for real estate. You can visualize a dataset as a table, with each object represented as a row and its attributes as columns. Datasets function as append-only storage, allowing for the addition of new records but preventing modifications or deletions of existing ones.
+
+Every Crawlee project run is linked to a default dataset, which is generally used to store the results specific to that crawler execution. Utilizing this dataset is optional.
+
+In Crawlee, datasets are represented by the Dataset class. To facilitate writing to the default dataset, Crawlee provides the `Dataset.pushData()` function.
+
+The data is stored in the directory specified by the `CRAWLEE_STORAGE_DIR` environment variable as follows:
+
+```text
+{CRAWLEE_STORAGE_DIR}/datasets/{DATASET_ID}/{INDEX}.json
+```
+- `{CRAWLEE_STORAGE_DIR}`: The root directory for all storage data specified by the environment variable.
+- `{DATASET_ID}`: Specifies the dataset, either "default" or a custom dataset ID.
+- `{INDEX}`: Represents the zero-based index of the record within the dataset.
+
+The following code demonstrates basic operations of the dataset:
+
+<Tabs groupId="dataset">
+    <TabItem value="dataset_basic_example" label="Basic usage" default>
+        <CodeBlock className="language-python">
+            {DatasetBasicExample}
+        </CodeBlock>
+    </TabItem>
+</Tabs>
```

Comment:
Why tabs if you have there only one item? That does not make sense. See how this is done in the [Request storage guide](https://crawlee.dev/python/docs/guides/request-storage).

Reply:
Same as for KVS.